capi-config/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+filegroup(
```

Comment:
Please ignore all `capi-config/*` files! They are copied in from go mod and there is no filter (yet) so we get everything.